core/electron/src/backend/ElectronHost.ts

```diff
@@ -105,7 +99,7 @@ export class ElectronHost {
   public static get electron() { return this._electron; }
 
   /** @internal */
-  public static get authorization() { return IModelHost.authorizationClient as ElectronAuthorizationBackend; }
+  public static get authorization() { return IModelHost.authorizationClient; }
```

Comment:
Is this used anywhere? If not, I'd be in favor of deleting it since it's the same as IModelHost.

Reply:
Doesn't seem to be. Removing.